product_stock_state/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,15 @@
+This module adds a "stock state" field on the product in order to inform the user of its general stock state at a glance.
+
+The state value can be :
+
+* In Stock
+* In Limited Stock (if qty available is under a threshold)
+* Resupplying (if qty forcasted is > 0)
+* Out of Stock (otherwise)
+
+The development of this module has been financially supported by:
+
+* Akretion R&D
+* Adaptoo
+* GRAP, Groupement Régional Alimentaire de Proximité <http://www.grap.coop>
```

Comment:
This part from 'The development of ...' should be put in CONTRIBUTORS file, I think
because it doesn't concern Usage but contribution.

Reply:
Right, after looking I just discovered that oca-gen-addon-readme has a CREDITS.rst which would be appropriate